sandbox/libs/authn/src/main/resources/shiro.ini

```diff
@@ -0,0 +1,29 @@
+# =======================
+# Shiro INI configuration
+# =======================
+
+[main]
+# Objects and their properties are defined here,
+# Such as the securityManager, Realms and anything
+# else needed to build the SecurityManager
+myRealm = org.opensearch.authn.realm.InternalRealm
+myRealm.realmName = myRealm
+myRealm.internalUsersYaml = example/example_internal_users.yml
+
+securityManager.subjectDAO.sessionStorageEvaluator.sessionStorageEnabled = false
+
+[users]
+# The 'users' section is for simple deployments
+# when you only need a small number of statically-defined
+# set of User accounts.
+
+[roles]
+# The 'roles' section is for simple deployments
+# when you only need a small number of statically-defined
+# roles.
+
+[urls]
+# The 'urls' section is used for url-based security
+# in web applications.  We'll discuss this section in the
+# Web documentation
+/** = noSessionCreation, authcBasic
```

Comment:
Do we need authcBasic here?

Reply:
Not currently, but this is an area we should look into further: https://shiro.apache.org/web.html

Removing this.